validator/src/cli.rs

```diff
@@ -1373,6 +1373,25 @@ pub fn app<'a>(version: &'a str, default_args: &'a DefaultArgs) -> App<'a, 'a> {
                 .possible_values(BlockProductionMethod::cli_names())
                 .help(BlockProductionMethod::cli_message())
         )
+        .arg(
+            Arg::with_name("wen_restart")
+                .long("wen-restart")
+                .value_name("DIR")
+                .takes_value(true)
+                .required(false)
+                .default_value(&default_args.wen_restart_path)
+                .conflicts_with("wait_for_supermajority")
+                .help(
```

Comment:
maybe something like 

"When specified, validators will enter Wen Restart mode which pauses normal activity. Validators in this mode will gossip their last voted fork with other validators participating in Wen Restart in order to achieve consensus on a safe restart slot. The safe restart slot will be a descendant of the latest optimistically confirmed slot to ensure we do not roll back any confirmed slots.

Progress for this mode will be saved in the file location provided, and if consensus is reached the validator will exit with the safe restart slot snapshot generated. At this point the operator can restart the node to enter wait_for_supermajority mode for the safe restart slot"

could also mention the repair procedure

Reply:
Changed, does this look better?

Reply:
ya looks good